packages/gestalt/src/Badge.js

```diff
@@ -1,24 +1,17 @@
 // @flow strict
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import cx from 'classnames';
 import styles from './Badge.css';
-import colors from './Colors.css';
 
 type Props = {|
-  position?: 'middle' | 'top',
```

Comment:
@denamwangi love the cleanup here! Since we are removing a prop, could you write a codemod for it? That way doing the update in the Pinterest codebase becomes easier.

https://github.com/pinterest/gestalt/tree/master/packages/gestalt-codemods contains a lot of examples and https://astexplorer.net/ is what you can use to test.

https://github.com/pinterest/gestalt/blob/master/packages/gestalt-codemods/1.35.0-1.36.0/leading-text-remove.js might be a good start, which converted
`<Text leading='short' />` to `<Text/>`
just like we will have to convert
`<Badge position="middle" />` to `<Badge/>`